app/views/dashboard/_alphabetical_collections_and_document_sets.html.slim

```diff
@@ -16,4 +16,4 @@
       -unless snippet.empty?
         p.collection_snippet =snippet
       .collection_summary
-        span =t('.owned_by', author: owner_link).html_safe
+        span =t('.project_by', author: owner_link).html_safe 
```

Comment:
I think this needs a bit of massaging.  You need nested parentheses:
(t('.project_by'), author: owner_link).html_safe  (probably!)

note how the "t" bit is a method that says "go look up the project_by string".

Test this by going to /collections on your test server or following the All Collections link at the bottom of the Find A Project screen.